sklearn/model_selection/tests/test_search.py

```diff
@@ -831,6 +843,175 @@ def test_search_iid_param():
         assert_almost_equal(train_std, 0)
 
 
+def test_grid_search_cv_results_multimetric():
+    X, y = make_classification(n_samples=50, n_features=4,
+                               random_state=42)
+
+    n_splits = 3
+    n_grid_points = 6
+    params = [dict(kernel=['rbf', ], C=[1, 10], gamma=[0.1, 1]),
+              dict(kernel=['poly', ], degree=[1, 2])]
+    scoring = ('accuracy', 'recall')
+    grid_search = GridSearchCV(SVC(), cv=n_splits, iid=False,
+                               param_grid=params, scoring=scoring,
+                               refit=False)
+    # TODO test for refit=True after review
+    grid_search.fit(X, y)
+    scoring = {'accuracy': make_scorer(accuracy_score),
+               'recall': make_scorer(recall_score)}
+    grid_search_iid = GridSearchCV(SVC(), cv=n_splits, iid=True,
+                                   param_grid=params, scoring=scoring,
+                                   refit=False)
+    grid_search_iid.fit(X, y)
+    # TODO test for refit=True after review
+
+    param_keys = ('param_C', 'param_degree', 'param_gamma', 'param_kernel')
+    score_keys = ('mean_test_accuracy', 'mean_train_accuracy',
```

Comment:
I think this test would be clearer by just doing:
`GridSearchCV(scoring=['accuracy', 'recall'], **kwargs)` and doing same for each of 'accuracy' and 'recall', and composing from the single metric searches what you expect the multimetric search result to look like (with exceptions for `*_time`).